src/elements/content-sidebar/MetadataSidebarRedesign.tsx

```diff
@@ -64,14 +64,7 @@ export interface MetadataSidebarRedesignProps extends PropsWithoutContext, Error
     api: API;
 }
 
-function MetadataSidebarRedesign({
-    api,
-    elementId,
-    fileId,
-    isBoxAiSuggestionsEnabled,
-    onError,
-    isFeatureEnabled,
-}: MetadataSidebarRedesignProps) {
+function MetadataSidebarRedesign({ api, elementId, fileId, onError, isFeatureEnabled }: MetadataSidebarRedesignProps) {
```

Comment:
is `isFeatureEnabled` doing what `useFeatureEnabled` is doing? do we need both?

Reply:
No I don't think so.

`useFeatureEnabled` is checking the `features` array. In our case we can check if metadata redesign is enabled, or if metadata aiSuggestions are enabled.

`isFeatureEnabled` is a prop that comes from host application in `metadataSidebarProps`
https://github.com/box/box-ui-elements/blob/549188a9f005b8dd0f63c0ed4906e8a64144c3a4/src/elements/content-sidebar/SidebarPanels.js#L61

it's defined here https://github.com/box/box-ui-elements/blob/549188a9f005b8dd0f63c0ed4906e8a64144c3a4/src/elements/content-sidebar/MetadataSidebar.js#L43-L47

maybe @bfoxx1906 or @jstoffan have more insight if it should be used 🤔

Reply:
Took a look, useFeatureEnabled isn't used anywhere else  in BUIE. However, Seems like a use case for components that don't have the features array passed down, so it probably is needed here. Have we tested useFeatureEnabled works as expected here?

Reply:
Yes, `useFeatureEnabled` works as expected. It's covered with a storybook story test (also in this PR). It was recommended by @jstoffan some time ago.

Now after re-reading this I understand initial confusion. As I mentioned `isFeatereEnalbed` is a prop indicating if user has __Metadata Sidebar__ enabled for them.